### PR TITLE
Fixes failing test case and adds update script

### PR DESCRIPTION
**Pull request by aaquibzama-tw**
This PR fixes the test case that was failing 'TestResultsReportsFailures' due to an earlier commit
It also adds curl update script 'update_talisman.bash' and related documentation

### CODE REVIEW

File: detector/filename_detector.go [hunk 1]
```diff
@@ -41,7 +41,6 @@ func DefaultFileNameDetector() Detector {
 		"^credentials\\.xml$",
 		"^.*\\.pubxml(\\.user)?$",
 		"^\\.?s3cfg$",
-		"^.*\\.ovpn$",
```
Comment (svishwanath-tw) [inline detector/filename_detector.go]: Why is this change a part of this PR ?

File: global_install_scripts/update_talisman.bash [hunk 2]
```diff
@@ -0,0 +1,142 @@
+#!/bin/bash
```
Comment (svishwanath-tw) [inline global_install_scripts/update_talisman.bash]: How is this file different from the basic installation script ?

Comment (aaquibzama-tw) [reply global_install_scripts/update_talisman.bash]: The install script downloads and sets up the talisman binary and hook script. It also sets up hooks in each git repository.

The update script only downloads the talisman binary and replaces it in the .talisman folder in the user home.

**Comment by svishwanath-tw** [comment]
Excellent work folks!

**Pull request merged**